x-pack/test_serverless/api_integration/test_suites/common/reporting/management.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { X_ELASTIC_INTERNAL_ORIGIN_REQUEST } from '@kbn/core-http-common/src/constants';
+import expect from '@kbn/expect';
+import { INTERNAL_ROUTES } from '@kbn/reporting-plugin/common/constants';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+// the archived data holds a report created by test_user
+const TEST_USERNAME = 'test_user';
+const TEST_USER_PASSWORD = 'changeme';
+const API_HEADER: [string, string] = ['kbn-xsrf', 'reporting'];
+const INTERNAL_HEADER: [string, string] = [X_ELASTIC_INTERNAL_ORIGIN_REQUEST, 'Kibana'];
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const reportingAPI = getService('svlReportingAPI');
+  const supertest = getService('supertestWithoutAuth');
+
+  describe('Reporting Management', function () {
+    const dataArchive = 'x-pack/test/functional/es_archives/reporting/archived_reports';
+
+    before(async () => {
+      await reportingAPI.createReportingRole();
+      await reportingAPI.createReportingUser();
+      await reportingAPI.createReportingUser(TEST_USERNAME, TEST_USER_PASSWORD);
+    });
+
+    beforeEach(async () => {
+      await esArchiver.load(dataArchive);
+    });
+
+    after(async () => {
+      await esArchiver.unload(dataArchive);
+    });
+
+    describe('Deletion', () => {
+      const DELETE_REPORT_ID = 'krazcyw4156m0763b503j7f9';
+
+      it(`user can delete a report they've created`, async () => {
+        const response = await supertest
+          .delete(`${INTERNAL_ROUTES.JOBS.DELETE_PREFIX}/${DELETE_REPORT_ID}`)
+          .auth(TEST_USERNAME, TEST_USER_PASSWORD)
+          .set(...API_HEADER)
+          .set(...INTERNAL_HEADER);
+
+        expect(response.status).to.be(200);
+        expect(response.body).to.eql({ deleted: true });
+      });
+
+      it(`user can not delete a report they haven't created`, async () => {
+        const response = await supertest
+          .delete(`${INTERNAL_ROUTES.JOBS.DELETE_PREFIX}/${DELETE_REPORT_ID}`)
+          .auth(reportingAPI.REPORTING_USER_USERNAME, reportingAPI.REPORTING_USER_PASSWORD)
```

Comment:
I'm honestly not sure how this test suite works, since it relies on custom users. Unfortunately, all of the stored reports in `dataArchive` rely on custom users. 🤷🏻

Reply:
Yeah I had to remove the calls to import security from FTR configs since it doesn't exist. I've left a lot of the code commented out but maybe when there's a clearer vision I can come back and implement the tests.